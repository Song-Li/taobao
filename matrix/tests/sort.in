SET b = sequence 3 -1.5
SET c = sorted b
SHOW c
