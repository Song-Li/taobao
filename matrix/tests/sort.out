> ok

> ok

> -1.50 0.00
1.50 3.00

> 
bye
